packages/compiler-cli/package.json

```
@@ -24,12 +24,12 @@
     "semver": "^6.3.0",
     "source-map": "^0.6.1",
     "sourcemap-codec": "^1.4.8",
-    "tslib": "^2.0.0",
+    "tslib": "^2.1.0",
     "yargs": "^16.1.1"
   },
   "peerDependencies": {
     "@angular/compiler": "0.0.0-PLACEHOLDER",
-    "typescript": ">=4.0 <4.2"
+    "typescript": ">=4.0 <4.3"
```

As discussed please set the lower limit to 4.2.3. There might be a few more places where this will need to be done.

```suggestion
    "typescript": ">=4.2.3 <4.3"
```